### PR TITLE
Fix typos in all docs

### PR DESCRIPTION
**Pull request by davydovanton**
:cat2:

### CODE REVIEW

File: doc/raketasks/backup_restore.md [hunk 1]
```diff
@@ -360,8 +360,8 @@ If you are using backup restore procedures you might encounter the following war
 
 ```
 psql:/var/opt/gitlab/backups/db/database.sql:22: ERROR:  must be owner of extension plpgsql
-psql:/var/opt/gitlab/backups/db/database.sql:2931: WARNING:  no privileges could be revoked for "public" (two occurences)
-psql:/var/opt/gitlab/backups/db/database.sql:2933: WARNING:  no privileges were granted for "public" (two occurences)
```
Comment (stanhu) [inline doc/raketasks/backup_restore.md]: Is this spelling coming from psql? If so, I think we should keep it the way it is.

Comment (davydovanton) [inline doc/raketasks/backup_restore.md]: I found only one `occurences` inclusions in psql repository ([link to github search](https://github.com/postgres/postgres/search?utf8=✓&q=occurences&type=Code)). That's why I didnt' think that this typo from psql.

**Comment by stanhu** [comment]
Thanks for proofreading, @davydovanton!

**Comment by davydovanton** [comment]
@stanhu thanks for review and merge! :tada:

**Pull request merged**